packages/data-stores/src/site/constants.ts

```diff
@@ -1 +1,3 @@
 export const STORE_KEY = 'automattic/site';
+
+export const PLACEHOLDER_SITE_ID = 211865921; // blankcanvas3demo.wordpress.com
```

Comment:
This constant is now duplicated in `client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/constants.ts` I think that copy is still used in `client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/pattern-assembler-container.tsx` but it would be nice to remove this duplication if possible